benchmarks/viscoelastic_sheared_torsion/viscoelastic_sheared_torsion.prm

```diff
@@ -125,6 +121,11 @@ subsection Material model
 
   subsection Visco Plastic
 
+    # Because we set the stress exponent n=1, the strain rate term of the dislocation creep
```

Comment:
Please add something like this before:
`We use the dislocation creep law to prescribe a constant viscous viscosity. `